Makefile.template

```diff
@@ -149,10 +151,13 @@ install_internal:
 	touch $(TIMESTAMP_INSTALL)
 
 runtime_tar_internal: $(TIMESTAMP_BUILD) install_internal
-	tar -zcf nnfw-package.tar.gz -C $(INSTALL_PATH) lib
-	tar -zcf nnfw-dev-package.tar.gz -C $(INSTALL_PATH) include/nnfw
-	tar -zcf nnfw-internal-dev-package.tar.gz -C $(INSTALL_PATH) include/onert
-	mv nnfw-*package.tar.gz $(INSTALL_PATH)/.
+	tar -zcf $(WORKSPACE)/nnfw-package.tar.gz -C $(INSTALL_PATH) lib
+	tar -zcf $(WORKSPACE)/nnfw-devel-package.tar.gz -C $(INSTALL_PATH) include/nnfw
+	tar -zcf $(WORKSPACE)/nnfw-plugin-devel-package.tar.gz -C $(INSTALL_PATH) include/onert
+	tar -zcf $(WORKSPACE)/nnfw-test-package.tar.gz -C ${INSTALL_PATH} bin tests test unittest unittest_standalone
```

Comment:
To run nnfw-test-package, we need to install nnfw-packge first ?

Reply:
@chunseoklee Yes.